src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -247,6 +249,7 @@ const EditHomepage = ({ match }) => {
         let highlightsErrors = []
         let announcementItemErrors = []
         const scrollRefs = []
+        const announcementScrollRefs = []
```

Comment:
for clarity here, we have htis because we want to scroll to each individual announcement instead of the section as a whole right?

Reply:
yes this is the case